database/migrations/2020_01_16_173600_create_opes_table.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use App\Traits\MigrationHelper;
+use Carbon\Carbon;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateOpesTable extends Migration
+{
+    use MigrationHelper;
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using short method names like CreateOpesTable::up(). The configured minimum method name length is 3.](https://app.codacy.com/manual/scriptotek/ub-baser/pullRequest?prid=4835963)